dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx

```diff
@@ -46,6 +44,7 @@ import {
   VisualizationType,
 } from '../../../../../common/types/custom_panels';
 import './visualization_flyout.scss';
+import { uiSettingsService } from '../../../../../common/utils';
```

Comment:
unused?

Reply:
This is a bug fix, the code using the import is already in [main](https://github.com/opensearch-project/trace-analytics/blob/cd516e5108156362a382ddca575fe228d7cf0223/dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx#L208).